### PR TITLE
[4.0] Table Column alignment (part 1) [a11y]

### PR DESCRIPTION
**Pull request by brianteeman**
com_associations
com_banners
com_cache
com_contacts
com_content
com_contenthistory

The alignment of columns has changed since J3 and also doesn't match the design for j4. In addition text-center alignment is not good for accessibility, particularly dyslexi8a (word-blindness).

https://www.w3.org/WAI/tutorials/tables/tips/
Alignment: Align text to the left and numeric data to the right (in left-to-right languages), so that people using larger text sizes or smaller screens will be able to find it. It’s helpful to give column headers the same alignment as the data in the cells below.

### CODE REVIEW

File: administrator/components/com_contact/tmpl/contacts/modal.php [hunk 1]
```diff
@@ -130,7 +130,7 @@
 						<td class="small d-none d-md-table-cell">
 							<?php echo LayoutHelper::render('joomla.content.language', $item); ?>
 						</td>
-						<td align="text-center">
+						<td class="text-center">
```
Comment (Quy) [inline administrator/components/com_contact/tmpl/contacts/modal.php:133]: Elsewhere, this column is `<td class="d-none d-md-table-cell">`. Should it be the same here?

Comment (brianteeman) [reply administrator/components/com_contact/tmpl/contacts/modal.php:133]: Honestly no idea what all those classes do. I just corrected the obvious error

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks - only another 60 files to go

**Pull request merged**